apps/myaccount/src/extensions/configs/profile.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ProfileConfig } from "./models";
+
+export const profileConfig: ProfileConfig = {
+    attributes: {
+        getRegExpValidationError: (attribute: string) => {
+            if (attribute === "phoneNumbers") {
```

Comment:
Get `phoneNumbers` from the contstants.

https://github.com/wso2/identity-apps/blob/69d11407f15f7c7a90861d04ee36f13e2fa946ae/modules/core/src/constants/profile-constants.ts#L62

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3043/commits/35ca0bc0741f40aec9fed064394f80d35148e926